datafusion/functions-array/src/kernels.rs

```diff
@@ -1,254 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I moved this code to `to_string.rs` to align with keeping the ScalarFunction definitions inline with their implementations